### PR TITLE
Cleanup, pt. 2

### PR DESCRIPTION
**Pull request by mimizh2418**
Minor code cleanup and less scuffed logging of active pathplanner path

### CODE REVIEW

**Review by rutmanz — CHANGES_REQUESTED** [review]
Oops wrong button

File: src/main/java/org/team1540/robot2024/RobotContainer.java [hunk 1]
```diff
@@ -141,20 +115,6 @@ public RobotContainer() {
                         new Indexer(
                                 new IndexerIOSim()
                         );
-
```
[the remark this answers is not in the document]
Comment (mimizh2418) [reply src/main/java/org/team1540/robot2024/RobotContainer.java:144]: They were already in drivetrain

File: src/main/java/org/team1540/robot2024/util/PathHelper.java [hunk 2]
```diff
@@ -19,16 +19,16 @@
 public class PathHelper {
     private static final PathConstraints constraints = new PathConstraints(
             3.0, 4.0,
-            Units.degreesToRadians(540), Units.degreesToRadians(720)
+            Constants.Drivetrain.MAX_ANGULAR_SPEED,
+            Constants.Drivetrain.MAX_ANGULAR_SPEED * (4.0/3.0)
```
[the remark this answers is not in the document]
Comment (mimizh2418) [reply src/main/java/org/team1540/robot2024/util/PathHelper.java:23]: I'm changing the path constraints to be defined by the actual limits of the robot (also yeah i forgot to do that for translation), is there some reason why we shouldn't do this?

[the remark this answers is not in the document]
Comment (rutmanz) [reply src/main/java/org/team1540/robot2024/util/PathHelper.java:23]: No why specifically 4/3

File: src/main/java/org/team1540/robot2024/RobotContainer.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (rutmanz) [reply src/main/java/org/team1540/robot2024/RobotContainer.java:144]: Even the sim specific stuff?

File: src/main/java/org/team1540/robot2024/util/PathHelper.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (mimizh2418) [reply src/main/java/org/team1540/robot2024/util/PathHelper.java:23]: The ratio before was 4/3 (720/540) so it just keeps the same acceleration time

File: src/main/java/org/team1540/robot2024/RobotContainer.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mimizh2418) [reply src/main/java/org/team1540/robot2024/RobotContainer.java:144]: AFAIK the sim specific stuff just makes pathplanner log the current pose and target pose as the same thing, which doesn't seem necessary

File: src/main/java/org/team1540/robot2024/util/PathHelper.java [hunk 3]
```diff
@@ -47,11 +47,11 @@ public static PathHelper fromPathPlannerPath(String pathname, boolean shouldRese
     }
 
 
-    private PathHelper(String pathname, boolean isChoreo, boolean shouldReset, boolean canFlip) {
+    private PathHelper(String pathname, boolean isChoreo, boolean shouldReset, boolean shouldFlip) {
         this.pathname = pathname;
         this.isChoreo = isChoreo;
         this.isResetting = shouldReset;
-        this.canFlip = canFlip;
+        this.shouldFlip = shouldFlip;
```
[the remark this answers is not in the document]
Comment (mimizh2418) [reply src/main/java/org/team1540/robot2024/util/PathHelper.java:54]: Is this variable even necessary? It's not being used anywhere and the flipping doesn't use it

**Review by rutmanz — APPROVED** [review]

**Review by rutmanz — APPROVED** [review]

**Pull request merged**